docker-entrypoint-funcs.sh

```diff
@@ -752,18 +762,11 @@ edbdocker_bootstrap_instance() {
   bootstrap_cmd=""
   bootstrap_opts=( "$@" )
 
-  if [ -n "${EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE}" ]; then
-    if ! [ -e "${EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE}" ]; then
-      edbdocker_die "ERROR: the file specified by EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE (${EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE}) does not exist."
-    else
-      bootstrap_opts+=(--bootstrap-script="${EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE}")
-    fi
-
-  elif [ -n "${EDGEDB_SERVER_BOOTSTRAP_COMMAND}" ]; then
+  if [ -n "${EDGEDB_SERVER_BOOTSTRAP_COMMAND}" ]; then
     bootstrap_opts+=(--bootstrap-command="${EDGEDB_SERVER_BOOTSTRAP_COMMAND}")
 
   elif [ -e "/edgedb-bootstrap.edgeql" ]; then
-    bootstrap_opts+=(--bootstrap-script="/edgedb-bootstrap.edgeql")
+    bootstrap_opts+=(--bootstrap-command-file="/edgedb-bootstrap.edgeql")
```

Comment:
You need to check if the server actually supports the new spelling with `edbdocker_server_supports`